modules/effects/src/actions.ts

```diff
@@ -19,17 +22,79 @@ export class Actions<V = Action> extends Observable<V> {
     observable.operator = operator;
     return observable;
   }
-
-  ofType<V2 extends V = V>(...allowedTypes: string[]): Actions<V2> {
```

Comment:
This may sound stupid, but here it goes anyways:
Since we have the fallback `ofType`, could we not remove this but mark it as deprecated?
With the next release we can remove it.
I say this because while the `compat` solution works, the user would have to change the import statement?

Reply:
I have no opinions on backwards compat (google operates on one-version policy), and will do whatever you guys decide.

Reply:
I was planning on doing a 6.1 release, so maybe we can deprecate the instance operators before release and then move forward with the removal for the next major release.